api/app/auto_spatial_advisory/elevation.py

```diff
@@ -315,7 +319,12 @@ def intersect_raster_by_firezone(threshold: int, advisory_shape_id: int, source_
     :param temp_dir: A temporary location for storing intermediate files
     """
     output_path = os.path.join(temp_dir, f"firezone_{source_identifier}_threshold_{threshold}.tif")
-    warp_options = gdal.WarpOptions(format="GTiff", cutlineDSName=DB_READ_STRING, cutlineSQL=f"SELECT geom FROM advisory_shapes WHERE id={advisory_shape_id}", cropToCutline=True)
+    warp_options = gdal.WarpOptions(
+        format="GTiff",
+        cutlineDSName=DB_READ_STRING,
+        cutlineSQL=f"SELECT geom FROM public.advisory_shapes WHERE id={advisory_shape_id}",
```

Comment:
This still works?

Reply:
Oh this isn't used anymore (it's from the old elevation stats job). There's a ticket to clean that up: https://github.com/bcgov/wps/issues/3997